.travis.yml

```diff
@@ -61,6 +66,8 @@ script:
       PX4_DOCKER_REPO=px4io/px4-dev-clang ./Tools/docker_run.sh 'make clang-tidy-parallel';
     elif [ "${TRAVIS_OS_NAME}" = "osx" ]; then
       make tests;
+    elif [[ "${BUILD_TARGET}" = "px4_clang" && "${TRAVIS_BRANCH}" != "coverity" ]]; then
+      PX4_DOCKER_REPO=px4io/px4-dev-nuttx-clang ./Tools/docker_run.sh 'make px4fmu-v2_default';
```

Comment:
@mayanez can you use a tag? That why the build system isn't dependent on potential docker breakage. px4io/px4-dev-nuttx-clang:2017-04-22